docs/docs/cmd/aad/administrativeunit/administrativeunit-add.mdx

```diff
@@ -0,0 +1,115 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# aad administrativeunit add
+
+Creates a new administrative unit
+
+## Usage
+
+```sh
+m365 aad administrativeunit add [options]
+```
+
+## Options
+
+```md definition-list
+`-n, --displayname <displayName>`
+: Display name for the administrative unit.
+
+`-d, --description [description]`
+: Description for the administrative unit.
+
+`--hiddenMembership [hiddenMembership]`
+: Indicates whether the administrative unit and its members are hidden.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be either **Global Administrator** or **Privileged Role Administrator**.
+
+:::
+
+## Examples
+
+Create an administrative unit with a specific display name
+
+```sh
+m365 aad administrativeunit add --displayName 'Marketing Division'
+```
+
+Create an administrative unit with a specific display name and description
+
+```sh
+m365 aad administrativeunit add --displayName 'Marketing Division' --description 'Marketing department administration'
+```
+
+Create a hidden administrative unit with a specific display name
+
+```sh
+m365 aad administrativeunit add --displayName 'Marketing Division' --hiddenMembership
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+  "id": "00b45a1b-7632-4e94-a3bd-f06aec976d31",
```

Comment:
Let's add some indentation to this json object

Reply:
added